src/fides/api/ops/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -463,6 +482,62 @@ def get_dataset(
     return dataset_config.ctl_dataset
 
 
+@router.get(
+    DATASET_CONFIGS,
+    dependencies=[Security(verify_oauth_client, scopes=[DATASET_READ])],
+    response_model=Page[DatasetConfigSchema],
+)
+def get_dataset_configs(
+    db: Session = Depends(deps.get_db),
+    params: Params = Depends(),
+    connection_config: ConnectionConfig = Depends(_get_connection_config),
+) -> AbstractPage[DatasetConfig]:
+    """Returns all Dataset Configs attached to current Connection Config."""
+
+    logger.info(
+        "Finding all dataset configs for connection '{}' with pagination params {}",
+        connection_config.key,
+        params,
+    )
+    dataset_configs = DatasetConfig.filter(
+        db=db, conditions=(DatasetConfig.connection_config_id == connection_config.id)
+    ).order_by(DatasetConfig.created_at.desc())
+
+    return paginate(dataset_configs, params)
```

Comment:
This is identical to `get_datasets` except the response has both the DatasetConfig fides_key and the CTL Dataset fides_key, which can differ.  The other parallel endpoint will eventually be deprecated.